locations/spiders/floorworld_au.py

```diff
@@ -0,0 +1,33 @@
+import chompjs
+from scrapy import Selector, Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class FloorworldAUSpider(Spider):
+    name = "floorworld_au"
+    item_attributes = {"brand": "Floorworld", "brand_wikidata": "Q117156913"}
+    allowed_domains = ["www.floorworld.com.au"]
+    start_urls = ["https://www.floorworld.com.au/store-finder"]
+
+    def parse(self, response):
+        locations = chompjs.parse_js_object(
+            response.xpath('(//div[contains(@class, "goto-next")]/following::script)/text()').get()
+        )["objects"]
+        for location in locations:
+            item = DictParser.parse(location)
+            item["ref"] = item["name"]
```

Comment:
Can you use something more stable than the name as `ref`? Maybe `location["hs_path"]` piece?

Reply:
Done, thanks.